pytradfri/aiocoap_api.py

```diff
@@ -0,0 +1,88 @@
+"""Coap implementation using aiocoap."""
+import json
+import logging
+
+import asyncio
+from aiocoap import Message, Context, GET, PUT
+from aiocoap.transports.tinydtls import PSK_STORE
+from pytradfri.command import Command
+from .parser import process_output
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def api_factory(host, security_code):
+    """Generate a request method."""
+
+    PSK_STORE[b'Client_identity'] = security_code.encode('utf-8')
```

Comment:
So this is how to auth using my fork. The real branch has evolved a bit more and we should make sure we fix the auth against that before merging.

Security is set as a property on the transport: https://github.com/chrysn/aiocoap/blob/tinydtls/aiocoap/transports/tinydtls.py#L182